<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhPreferUppercase</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color:rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="create">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>Cria Tabelas</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoLista">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataCon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="insert">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>Realiza Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="excel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>Excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update_lista">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>CTR LISTA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drop">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>Drop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update_ncm">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>NCM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update_ms">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>REGISTRO MS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update_cest">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>CEST</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoCest">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoMs">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoNcm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoInsert">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoCreate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoDrop">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoExcel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="insertNcm">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 5px;
	color:rgb(255, 255, 255);
	background-color: rgb(0,0,0);

}

QPushButton:hover {
	color:rgb(255, 255, 255);
	background-color: rgb(68, 68, 68)	;

}
</string>
    </property>
    <property name="text">
     <string>Realiza Insert NCM</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfoInsertNcm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(208, 39, 24)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
